beta/components/header/index.js

```diff
@@ -167,7 +167,7 @@ class Header extends Component {
     this.local.resolved = props.resolved
     this.local.href = props.href
 
-    const nav = () => {
+    const mainMenu = () => {
```

Comment:
Renamed this to avoid ambiguity, it's not just a nav that is rendered all the time, but specifically the "main one" (open to other names).